### ISSUE TITLE
Update issue reference in 2.4.3 release notes

### ISSUE THREAD

**Issue by ak77-prime** [magento/devdocs#9253]
# General issue

## Description:

https://devdocs.magento.com/guides/v2.4/release-notes/commerce-2-4-3.html#payment-methods

Under the ``Paypal`` subheading, the page source indicates:

```
<!--- MC-32805-->

<ul>
  <li>Shoppers are now redirected to the shopping cart page instead of the product page when they try to update the mini cart for an order that contains deleted items. Previously, shoppers were redirected to the product page.</li>
</ul>
```

However, MC-32805 refers to a totally different issue: ``Error message appears when update qty item in the cart.``

[Per discussion with our QA Engineer in Slack](https://magento.slack.com/archives/G01CSAGHZEJ/p1637128506063600):

```
I don’t see these changes https://github.com/magento-l3/magento2ce/pull/193/files on a 2.4.3 installation. So a backport is needed
```

## Possible solutions:

Either refer to the correct MC link (however, per above, it's not fixed in 2.4.3) or remove the above "Fix" from the release notes [more likely]

It looks like it could be https://jira.corp.magento.com/browse/MC-42442 - however, it appears that the fix wuold only be included in 2.4.4

```
Fix version
2.4.4
Short Description
PayPal Payments Advanced payment method does not redirect to the success page after the payment has been processed
```

**Comment by mimou78** [comment]
Hello @ak77-prime,

 I have just checked in the link of the documentation that you provided I do not see the mention of <! --- MC-32805 ->

**Comment by ak77-prime** [comment]
Look at the **Page Source/HTML**

**Comment by keharper** [comment]
Also check whether the Open Source release notes need to be updated.

**Comment by jfrontain** [comment]
hi @ak77-prime, thanks for catching this. The Jira ticket indicated that this fix was merged into 2.4.3, but it was actually merged into 2.4.4. I've removed the issue description from the  2.4.3 Adobe Commerce and Magento Open Source notes, and the Jira ticket has been updated.
